inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -569,6 +577,218 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         return res;
     });
 
+    addSpecificCreator({"PRelu"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "PReLU",
+                             details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<PReLULayer>(attrs);
+        res->params = params;
+
+        Builder::NodeConverter<ngraph::op::Constant> converter;
+        const auto weightsNode = node->input_value(1).get_node_shared_ptr();
+        if (converter.canCreate(weightsNode)) {
+            const auto& weights = converter.createLayer(weightsNode);
+            res->blobs["weights"] = weights->blobs["custom"];
+            res->_weights = weights->blobs["custom"];
+        }
+        return res;
+    });
+
+    addSpecificCreator({"TileIE"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "Tile",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<TileLayer>(attrs);
+        res->params = params;
+        return res;
+    });
+
+    addSpecificCreator({"PriorBoxIE"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "PriorBox",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto parseBoolStrToIntStr = [](const std::string &param) -> const std::string {
+            if (param == "true") {
+                return "1";
+            }
+            else if (param == "false") {
+                return "0";
+            }
+            return param;
+        };
```

Comment:
there should be similar function already. 

@ilyachur can you provide it?

Reply:
Even if it does not exist then it should be placed to some common place.

Reply:
Such method doesn't exist. But from my point of view we can add something like: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/legacy_api/include/legacy/ie_layers.h#L336

Reply:
I agree on this, I've just added such function, but I'm not sure about it's placement, can you take a look?

Reply:
I think you can extend CNNLayer methods. It is a legacy API and we use it only to convert Ngraph to legacy API

Reply:
Added a CNNLayer method for rhat, I'm going to resolve this conversation